docs/en/observability/whats-new.asciidoc

```diff
@@ -22,9 +22,148 @@ https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
 // tag::whats-new[]
 
 [discrete]
-== APM failed transactions correlation charts and top values (Beta)
+== Turnkey data onboarding from AWS Fargate with FireLens
 
-In 7.16, the failed transaction correlations include a new chart, which shows
+We are excited to announce the first native integration with Amazon Web Services (AWS) on {ecloud}, via AWS FireLens, a container log router for Amazon EC2 and AWS Fargate. You can now quickly and easily ingest container logs into {ecloud} within seconds using FireLens and Fluent Bit without needing to install and manage standalone data shippers (see https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/elastic-cloud[amazon-ecs-firelens-examples]). The simplified data architecture also reduces operational complexities.
+
+[role="screenshot"]
+image::images/aws-integration.png[AWS integration]
+
+[discrete]
+== Unified integrations
+
+With 7.16 it is now much easier to discover the 200+ integrations we offer and use them to onboard data, regardless of the underlying technology behind the integrations.
+
+[role="screenshot"]
+image::images/integrations-aws.png[Integrations]
+
+[discrete]
+== Key Observability integrations are now GA
+
+You can now start using {agent} integrations for the majority of key Observability use cases, with the benefits of central management, scalability, security, and one-click integration.
+
+Integrations which are now generally available (GA) include:
+
+* GCP
+* Docker, Kubernetes, Zookeeper
+* Apache, Apache Tomcat
+* Cassandra, MongoDB, MySQL, Postgres, Redis
+* NATS, RabbitMQ
+* Nginx, Nginx Ingress Controller, Traefik
+* STAN
+
+[discrete]
+== Self-managed Elastic Package Registry is now GA
+
+We are pleased to announce the general availability (GA) of our self-managed Elastic Package Registry (EPR), an OCI compliant Docker image. The Elastic Stack needs to understand the external data sources and available data formats to integrate, collect, and visualize data using Elastic Agents. Today, {kibana} receives this knowledge in the form of packages from a SaaS EPR endpoint. Self-managed EPR is a standalone server that will enable the distribution of integration packages just like the existing SaaS EPR in any infrastructure setting of your choice. We know you wanted to ensure that each component of your Elastic Stack is compliant with your organization's policies, which may include:
+
+* Operating in an air-gapped environment
+* Not adopting software unless it is officially supported
+* Discarding any software that cannot use custom certificates for communications
+
+To remove all the hurdles from your integration journey with {fleet} server and {agent}, we are promoting our self-managed EPR to an officially supported GA status. It can now use your custom certificates to talk to {kibana} and provide the out-of-box assets and documentation for all your desired integrations.
```

Comment:
```suggestion
To remove all the hurdles from your integration journey with {fleet} server and {agent}, we are promoting our self-managed EPR to an officially supported GA status. It can now use your custom certificates to talk to {kib} and provide the out-of-box assets and documentation for all your desired integrations.
```